create-reporter/index.js

```diff
@@ -197,12 +197,12 @@ const REPORTERS = {
 
 function createReporter (options) {
```

Comment:
Maybe it's also worth to split this function something like this
```
function createLogger (options) {
  if (typeof options.logger === 'object') {
    return options.logger
  } else {
    let stream = options.reporterStream || pino.destination()
    let prettifier = {}
    if (options.logger === 'human') {
      let env = options.env || process.env.NODE_ENV || 'development'
      let color = env !== 'development' ? false : undefined
      prettifier = {
        prettyPrint: {
          basepath: options.root,
          color
        },
        prettifier: humanFormatter
      }
    }
    return pino(
      {
        name: 'logux-server',
        timestamp: pino.stdTimeFunctions.isoTime,
        ...prettifier
      },
      stream
    )
  }
}

function createReporter (options) {
  function reporter (type, details) {
    let report = REPORTERS[type](details)
    let level = report.level || 'info'
    reporter.logger[level](report.details || details || {}, report.msg)
  }
  reporter.logger =
    typeof options.logger === 'object' ? options.logger : createLogger(options)
  return reporter
}
```

Reply:
Yeap, let’s split it too